catalog/lib/theme/app_buttons.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:catalog/theme/app_text_styles.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:catalog/theme/app_colors.dart';
+
+typedef AppFilledButton = FilledButton;
+typedef AppStrokeButton = OutlinedButton;
```

Comment:
What do you think about using the name directly instead of adding App? 
```suggestion
typedef StrokeButton = OutlinedButton;
```